### PR TITLE
DB::repair: pass Options by ref

### PR DESCRIPTION
**Pull request by ordian**
This is a breaking change.

### CODE REVIEW

**Comment by ordian** [comment]
CI failure seems spurious.

File: src/db.rs [hunk 1]
```diff
@@ -899,7 +899,7 @@ impl DB {
         Ok(())
     }
 
-    pub fn repair<P: AsRef<Path>>(opts: Options, path: P) -> Result<(), Error> {
```
Comment (aleksuss) [inline src/db.rs]: What is a point ? Do we need to use `Options` after repairing ?

Comment (ordian) [reply src/db.rs]: Yes, that's exactly what we do:
```rust
let db_corrupted = Path::new(path).join(Database::CORRUPTION_FILE_NAME);
if db_corrupted.exists() {
	DB::repair(&opts, path)?;
	//         ^^^^^
}
// ...
DB::open_cf(&opts, path, &cfnames)
```

**Comment by vitvakatu** [comment]
No reason not to accept something via ref if it's possible.

**Pull request merged**